src/test/java/seedu/address/model/date/TrainingDateTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.model.date;
```

Comment:
Good job creating tests for the method implemented!

Reply:
thanks! lets try to maintain our coverage